fenix/app/src/main/java/org/mozilla/fenix/debugsettings/ui/DebugDrawerLandingScreen.kt

```diff
@@ -0,0 +1,129 @@
+
+package org.mozilla.fenix.debugsettings.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.Snackbar
+import androidx.compose.material.SnackbarHost
+import androidx.compose.material.SnackbarHostState
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+import mozilla.components.support.ktx.android.content.appName
+import mozilla.components.support.ktx.android.content.appVersionName
+import org.mozilla.fenix.compose.Divider
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.inComposePreview
+import org.mozilla.fenix.compose.list.TextListItem
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * The landing screen of the [DebugDrawer].
+ *
+ * @param menuItems The list of [DebugDrawerMenuItem]s.
+ */
+@Composable
+fun DebugDrawerLandingScreen(
+    menuItems: List<DebugDrawerMenuItem>,
+) {
+    val lazyListState = rememberLazyListState()
+
+    val appName: String
+    val appVersion: String
+    if (inComposePreview) {
+        appName = "App Name Preview"
+        appVersion = "100.00.000"
+    } else {
+        appName = LocalContext.current.appName
+        appVersion = LocalContext.current.appVersionName
+    }
+
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = FirefoxTheme.colors.layer1),
+        state = lazyListState,
+    ) {
+        item {
+            Row(
+                modifier = Modifier
+                    .padding(all = 16.dp)
+                    .fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+            ) {
+                Text(
+                    text = appName,
+                    color = FirefoxTheme.colors.textPrimary,
+                    style = FirefoxTheme.typography.headline5,
+                )
+
+                Text(
+                    text = appVersion,
+                    color = FirefoxTheme.colors.textSecondary,
+                    style = FirefoxTheme.typography.headline5,
+                )
+            }
+
+            Divider()
+        }
+
+        items(menuItems) { menuItem ->
```

Comment:
💡 It would be performant to have an `id` and set that as key for the each item --> https://developer.android.com/jetpack/compose/performance/bestpractices#use-lazylist-keys

Reply:
good catch